### PR TITLE
Update copyright

### PR DESCRIPTION
**Pull request by atigyi**
Updating the copyright years to include 2020. The .html dox will be regenerated in a separated PR.

### CODE REVIEW

File: doc/porting_guide.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Google Cloud IoT Device SDK for Embedded C Porting Guide
-##### Copyright 2018-2019 Google LLC
+
```
Comment (DellaBitta) [inline doc/porting_guide.md]: nit: Looks like a space snuck in here

File: examples/zephyr_native_posix/prebuild.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google LLC
+# Copyright 2018-2020 Google LLC
```
Comment (DellaBitta) [inline examples/zephyr_native_posix/prebuild.sh]: 2019-2020

File: examples/zephyr_native_posix/src/main.c [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Google LLC
+/* Copyright 2018-2020 Google LLC
```
Comment (DellaBitta) [inline examples/zephyr_native_posix/src/main.c]: 2019-2020

File: make/mt-config/mt-presets.mk [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Google LLC
+# Copyright 2018-2020 Google LLC
```
Comment (DellaBitta) [inline make/mt-config/mt-presets.mk]: 2019-2020

File: src/bsp/crypto/wolfssl/iotc_bsp_crypto_wolfssl.h [hunk 5]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Google LLC
+/* Copyright 2018-2020 Google LLC
```
Comment (DellaBitta) [inline src/bsp/crypto/wolfssl/iotc_bsp_crypto_wolfssl.h]: 2019-2020

File: src/bsp/platform/zephyr/iotc_bsp_io_net_zephyr.c [hunk 6]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Google LLC
+/* Copyright 2018-2020 Google LLC
```
Comment (DellaBitta) [inline src/bsp/platform/zephyr/iotc_bsp_io_net_zephyr.c]: 2019-2020

File: src/bsp/platform/zephyr/iotc_bsp_mem_zephyr.c [hunk 7]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Google LLC
+/* Copyright 2018-2020 Google LLC
```
Comment (DellaBitta) [inline src/bsp/platform/zephyr/iotc_bsp_mem_zephyr.c]: 2019-2020

File: src/bsp/platform/zephyr/iotc_bsp_rng_zephyr.c [hunk 8]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Google LLC
+/* Copyright 2018-2020 Google LLC
```
Comment (DellaBitta) [inline src/bsp/platform/zephyr/iotc_bsp_rng_zephyr.c]: 2019-2020

File: src/bsp/platform/zephyr/iotc_bsp_time_zephyr.c [hunk 9]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019 Google LLC
+/* Copyright 2018-2020 Google LLC
```
Comment (DellaBitta) [inline src/bsp/platform/zephyr/iotc_bsp_time_zephyr.c]: 2019-2020

File: third_party/mqtt-protocol-c/LICENSE.md [hunk 10]
```diff
@@ -1,4 +1,4 @@
-# 2018-2019Google LLC
+# 2018-2020 Google LLC
```
Comment (DellaBitta) [inline third_party/mqtt-protocol-c/LICENSE.md]: :+1:

File: doc/porting_guide.md (hunk 1 above)
Comment (atigyi) [reply doc/porting_guide.md]: the new line?

Comment (atigyi) [reply doc/porting_guide.md]: was added intentionally, to be similar to its company a file user_guide.md, next to it

**Pull request merged**